### PR TITLE
Auth 2184/project limit config docs

### PR DESCRIPTION
**Pull request by blakestier**
### :nut_and_bolt: Description: What code changed, and why?

We now allow customers to modify the maximum number of projects that can be created. This documents the steps to do so.

### CODE REVIEW

File: components/automate-chef-io/content/docs/iam-v2-guide.md [hunk 1]
```diff
@@ -239,6 +239,21 @@ Projects can be managed via the Projects list under the **Settings** tab and con
 [assigning ingested resources to projects]({{< relref "iam-v2-guide#assigning-ingested-resources-to-projects" >}}),
 so they are not relevant when assigning IAM resources such as teams or roles.
 
+#### Configuring Project Limit
+
+By default, Chef Automate is currently limited to six projects. If you would like to increase that limit, you will need to do so using the Chef Automate cli:
```
Comment (susanev) [inline components/automate-chef-io/content/docs/iam-v2-guide.md]:
```suggestion
By default, Chef Automate is currently limited to six projects. If you would like to increase that limit, you will need to do so using the Chef Automate CLI:
```

File: components/automate-chef-io/content/docs/iam-v2-guide.md [hunk 2]
```diff
@@ -239,6 +239,21 @@ Projects can be managed via the Projects list under the **Settings** tab and con
...
+
+```
+cat << EOF > autht.toml
```
Comment (susanev) [inline components/automate-chef-io/content/docs/iam-v2-guide.md]: should this be `authz`?

File: components/automate-chef-io/content/docs/iam-v2-guide.md [hunk 3]
```diff
@@ -239,6 +239,21 @@ Projects can be managed via the Projects list under the **Settings** tab and con
...
+[auth_z.v1.sys.service]
+project_limit = <desired-max-projects>
+EOF`
```
Comment (susanev) [inline components/automate-chef-io/content/docs/iam-v2-guide.md]: is ` needed?

File: components/automate-chef-io/content/docs/iam-v2-guide.md [hunk 4]
```diff
@@ -239,6 +239,21 @@ Projects can be managed via the Projects list under the **Settings** tab and con
...
+By default, Chef Automate is currently limited to six projects. If you would like to increase that limit, you will need to do so using the Chef Automate CLI:
+
+```
+cat << EOF > authz.toml
+[auth_z.v1.sys.service]
+project_limit = <desired-max-projects>
+EOF
+```
+
+`chef-automate config patch authz.toml`
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/iam-v2-guide.md]: Is this line supposed to be the second step in committing the changes? An additional line explaining that may be needed between the file modification above and this line.
For example:

> Then, run `chef-automate configure patch authz.toml` to update the existing Chef Automate configuration.

**Pull request merged**